src/Arachne.cc

```diff
@@ -1245,10 +1192,10 @@ void descheduleCore() {
     // hold it for too long.
     // If this creation fails, it would implies that we are overloaded and
     // should not ramp down.
-    if (createThreadOnCore(minIndex, releaseCore) == NullThread) {
+    if (createThreadOnCore(corePolicy->baseClass, minCoreId, releaseCore) == NullThread) {
```

Comment:
Information leakage: why is Arachne referencing corePolicy->baseClass? Let's discuss.